backend/tracim_backend/lib/utils/utils.py

```diff
@@ -161,14 +161,28 @@ def get_timezones_list() -> List[Timezone]:
     return tz_list
 
 
-# INFO - G.M - 2018-08-02 - Simple password generator, inspired by
-# https://gist.github.com/23maverick23/4131896
-
+def generate_autogen_password_char() -> str:
+    """Return list of allowed characters for autogenerated password"""
+    alphanumeric_chars = string.ascii_letters + string.digits
+    authorized_special_characters = "!#%@+=?"
+    confusing_characters = "0Oo" + "1lI"
+    return str(
+        "".join(
+            [
```

Comment:
The list creation is not necessary here, join takes an iterable as parameter (and a list comprehension is iterable).

Reply:
updated.